elixir/apps/domain/lib/domain/auth/adapters/mock/jobs/sync_directory.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Domain.Auth.Adapters.Mock.Jobs.SyncDirectory do
```

Comment:
Just want to make sure I've not overlooked something.  This means every minute this adapter is creating all new users, groups, and group memberships for this provider and updating the DB?

Reply:
The `provider_identifier` is consistent, so it _should_ upsert them

Reply:
Gotcha....but their first/last names are being changed along with their email correct?  Is this just to try and stress test?

Reply:
Actually, nevermind, reading your answer to my other question kind of answered this one